### PR TITLE
Added references to the rules: rules_databricks, rules_k8s_extension and rules_microsoft_azure

### PR DESCRIPTION
**Pull request by jullianoacqio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -416,6 +424,7 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
       <td>
         <ul>
           <li><a href="https://github.com/bazelbuild/rules_k8s">bazelbuild/rules_k8s</a></li>
+          <li><a href="https://github.com/acqio/rules_k8s_extensions">acqio/rules_k8s_extensions</a> Bazel rules for Kubernetes extension for creating configmaps and secrets.</li>
```
Comment (jin) [inline README.md:427]: This link is 404 for me

**Review by jin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jullianoacqio) [reply README.md:427]: Oh my God. There is an extra S.

**Review by jin — APPROVED** [review]
Thank you!

File: README.md [hunk 2]
```diff
@@ -488,6 +497,15 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
         </ul>
       </td>
     </tr>
+    <tr>
```
Comment (jin) [inline README.md:500]: Extra `<tr>`?

Comment (jullianoacqio) [reply README.md:500]: adjusted.

**Pull request merged**